README.md

```diff
@@ -0,0 +1,143 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  # Newton's Capstone Project - Devspark
+
+</div>
+
+## 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [🎥 Video Walkthrough](#video-walkthrough)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+This is my portfolio project and it is based on this [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
+
+**[Newton-Portfolio]** is a HTML CSS and JS project
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML
+- CSS
+- Js
+- Linters
+- Figma
+
+### Key Features <a name="key-features"></a>
+- **Understanding Git, GitHub and GitHub Workflow**
+- **Configuring Linters**
+- **Changing design template to code**
+- **Creating mobile responsive portfolio website**
+- **Interacting with the DOM**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+[Devspark Website](https://newtonombese.github.io/Newton-Capstone-Project/)
```

Comment:
- [ ] Your project demo is not working well great job 😢  Please fix it and make all the pages work as expected in order to make your demo work as expected and your readme more professional 👍

Reply:
Fixed